src/Options/StdParsers.hpp

```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <complex>
```

Comment:
to reduce the potential for sucking in everything from the STL, please rename this file to `StdComplex` so we don't end up with a mega-file like StdHelpers